src/main/cpp/t05_minodd.cpp

```diff
@@ -28,5 +28,42 @@
 using namespace std;
 
 int t05_minodd() {
+	int min = 0;
+	bool m = 0;
+	bool s = 0;
+	int n;
+	cin >> n;
+	int mas[1000];
+	for (int i = 0; i < n; i++) {
+		cin >> mas[i];
+		if (((mas[i] % 2 == 1)||(mas[i] % 2 ==-1)) && (m == 0))
+		{
+			min = mas[i];
+			m = 1;
+		}
+	}
+	for (int i = 1; i < n; i++)
+	{
+
+		if (mas[i] < min && mas[i] % 2 == 1)
+		{
+			min = mas[i];
+			s ++;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incrementing a variable of type 'bool' with postfix operator++ is deprecated by the C++ Standard. You should assign it the value 'true' instead.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4532058)